File: ios/TaskCal/LiquidGlassFABViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(LiquidGlassFABViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(onPress, RCTBubblingEventBlock)
RCT_EXPORT_VIEW_PROPERTY(primaryColor, UIColor)
@end

File: ios/TaskCal/LiquidGlassFABViewManager.swift
import UIKit
import React

@objc(LiquidGlassFABViewManager)
class LiquidGlassFABViewManager: RCTViewManager {
    override func view() -> UIView! {
        if #available(iOS 26.0, *) {
            return LiquidGlassFABView()
        }
        return UIView()
    }
    override static func requiresMainQueueSetup() -> Bool { return true }
}
